acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-with-resolution-statuses.feature

```diff
@@ -0,0 +1,61 @@
+# For MODORDERS-1170
+Feature: Close order if order line has resolution statuses that should make it as closed
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': '*/*' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId = callonce uuid3
+    * def poLineId = callonce uuid4
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+  @Positive
+  Scenario: Oper order with resolution statuses that should make it as closed
+    # 1. Create fund and budget
+    * configure headers = headersAdmin
+    * def v = call createFund { id: #(fundId) }
+    * def v = call createBudget { id: #(budgetId), fundId: #(fundId), allocated: 100 }
+    * configure headers = headersUser
+
+    # 2. Create an order
+    * def v = call createOrder { id: #(orderId) }
+
+    # 3. Create an order line
+    * def v = call createOrderLine { id: #(poLineId), orderId: #(orderId), fundId: #(fundId), paymentStatus: 'Payment Not Required', receiptStatus: 'Receipt Not Required' }
+
+    # 4. Open the order
+    * def v = call openOrder { orderId: #(orderId) }
+
+    # 5. Check the order was closed, because paymentStatus and receiptStatus are 'Awaiting Payment' and 'Awaiting Receipt'
+    Given path 'orders/composite-orders', orderId
+    When method GET
+    Then status 200
+    And match $.workflowStatus == 'Closed'
```

Comment:
better to use `response` instead of `$` I think, much better to read and understand

Reply:
thanks improved all comments